README.md

```diff
@@ -113,22 +113,13 @@ We'll organise into different streams here. One kind request for motivation: Ple
 
 ### Spring Stack
 
-* Library Managemnet App
-* Authorization Server
-* Chatting App
-* eWallet App
-* Online Store App
-
-## Back End Ideas
-
-### Spring Stack
-
```

Comment:
Is it intentional deleting all these?

Reply:
Yes, I have deleted the in intentionally for duplicate names. Spring stack is present 2 times.